File: DevExpress-Demo/VB/GridDemo.Wpf/ModuleResources/PrintCardViewViewModel.vb
Namespace GridDemo

    Public Class PrintCardViewViewModel
        Inherits PrintViewModelBase

        Public Property UseCustomPrintStyles As Boolean
            Get
                Return GetProperty(Function() Me.UseCustomPrintStyles)
            End Get

            Set(ByVal value As Boolean)
                SetProperty(Function() UseCustomPrintStyles, value)
            End Set
        End Property

        Public Overrides Sub ReOpenPreviewTabs()
            MyBase.ReOpenPreviewTabs()
            UseCustomPrintStyles = True
            ShowPreviewInNewTab()
            UseCustomPrintStyles = False
        End Sub

        Protected Overrides Function GetTitle() As String
            Return If(UseCustomPrintStyles, "Uniform Cards Size", "Default") & " Style Print Preview"
        End Function
    End Class
End Namespace

File: DevExpress-Demo/CS/GridDemo.Wpf/ModuleResources/PrintCardViewViewModel.cs
using System;
using DevExpress.Mvvm.DataAnnotations;
using DevExpress.Xpf.Printing;

namespace GridDemo {
    public class PrintCardViewViewModel : PrintViewModelBase {
        public  bool UseCustomPrintStyles {
            get { return GetProperty(() => UseCustomPrintStyles); }
            set { SetProperty(() => UseCustomPrintStyles, value); }
        }

        public override void ReOpenPreviewTabs() {
            base.ReOpenPreviewTabs();
            UseCustomPrintStyles = true;
            ShowPreviewInNewTab();
            UseCustomPrintStyles = false;
        }

        protected override string GetTitle() {
            return (UseCustomPrintStyles ? "Uniform Cards Size" : "Default") + " Style Print Preview";
        }
    }
}
